Quantum-Control-Applications/Superconducting/Single Fixed Transmon/Use Case 3 - DRAG calibration/README.md

```diff
@@ -0,0 +1,75 @@
+
+## The device
+
+The chip contains 5 2D-transoms, where some of them have nearest-neighbor connectivity, 
+with flux tunable lines. For this experiment, we focus only in one transmon with a drive
+to its flux line to put the transmon  in the **sweep-spot**. All other transmons are left
+unbiased and signal is send to them.
+
+## Method and results
+
+We will show the determination of the DRAG coefficient with two different pulse sequences.
+The first one is related to the application of `(x180)(-x180)` train of pulses and the second
+is a sequence of `x180y90` and `y180x90` pulses. We are calibrating the `$\alpha$` parameter in 
+the following formula,
+
+<img src="https://latex.codecogs.com/svg.image?\Omega^{'} = \Omega + i\alpha\frac{d\Omega/dt}{\Delta}" />
+
+### DRAG coefficient calibration with `x180` `-x180` pulses
+
+In this step we generate a 2D plot where the axis are `number of (x180)(-x180)` pulses
+that applied, and the other axis is the `DRAG coefficient`. We gain sensitivitiy 
+to pinpoint to the DRAG coefficient by applying a train of pulses because
+which each extra pair of pulses there is more accumulation of population in the second
+excited state. The pattern generated in the image below resembles a `Chevron-Ramsey`
+experiment. We can identify that the center line is away from the zero value. We get
+a rough estimate by drawing a horizontal line at `0.1`. It is possible to do an x-axis
+line cut the image below and do a fitting around `0.1` to get a more precise value.
+To generate this plot we used the `coarse_DRAG.py`. The pulse train used is a 2D
+extension of the one listed in Ref. [].
+
+![Google_drag](Google_DRAG.png)
+
+### DRAG coefficient calibration with `x180y90` and `y180x90` pulses
+
+The figure below shows two lines that intersect at the value of the calibrated `DRAG coefficient`.
+This sequence is made of `x180y90` and `y180x90` pulses with one iteration of each only, not alike
+the in the previous method. As noted in Ref. [] this sequence can be used to calibrate the 
+DRAG coefficient, and it is an extension of `AllXY`. By fitting the lines and computing their
+intersection from the fitted formulas we can precisely derive the `DRAG coefficient`. The derived
+DRAG value is ~`0.11`.
+
+Note that we calibrate the mixer of the qubit drive for this demonstration. The frequency bandwidth
+of the `16 ns` pulse was ~ `100 MHz` and because the `IF = -95 MHz` for this experiment, in 
+the frequency domain the pulse overlaps with the LO leakage.
+
+![Reeds_drag](Reeds_DRAG.png)
+
+### Randomized benchmarking with and without DRAG and AC-stark shift correction
+
+RB is a technique used to characterize the Fidelity of qubit gates. In this case we are
+focusing in the gates for a single qubit. The figure below shows three decaying curves 
+that end in the `mixed state` due to the depolarization occurring during application of pulses
+to the qubit. Having corrected for leakage out of the computational space we find a
```

Comment:
The decay is also because of T1, so maybe I would write:
... end in the `mixed state` due to the thermal relaxation and the depolarization occurring during the application of pulses
to the qubit

Reply:
thanks